### PR TITLE
Add  custom training bq managed dataset notebook

### PR DESCRIPTION
**Pull request by ivanmkc**
Added example for training a custom model using BQ data.

Also, added a ignore for E203 as flake8 doesn't play nice with black otherwise.

### CODE REVIEW

File: ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 1]
```diff
@@ -0,0 +1,1560 @@
+{
```
Comment (thedriftofwords) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Change to Vertex AI throughout</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (thedriftofwords) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Put the names of the fields in code font.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (thedriftofwords) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>What version of TF is being used in this tutorial? Do we still need the TF 2.3 note?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (thedriftofwords) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Use sentence case for all headings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>I think we can assume the user is using the latest version of TF, which we test for.</p>

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: Done

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: Done

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: Done

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: Done

File: ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 2]
```diff
@@ -0,0 +1,1545 @@
...
```
Comment (thedriftofwords) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>In this version of the dataset, you will use...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (thedriftofwords) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>I commented on this section in another PR - see https://github.com/GoogleCloudPlatform/ai-platform-samples/pull/381#discussion_r658193573</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (thedriftofwords) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Fix this section title (typo, sentence case, product name)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/356/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Done? I think.</p>

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Done</p>

Comment (ivanmkc) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Done</p>

Comment (thedriftofwords) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:1]: <p>Looks better, thanks!</p>

**Review by thedriftofwords — APPROVED** [review]

**Comment by ivanmkc** [comment]
@thedriftofwords Thanks, can I get a stamp if ready?

**Comment by ivanmkc** [comment]
Thanks!

**Pull request merged**